pkg/nsx/services/vpc/vpc.go

```diff
@@ -997,8 +997,8 @@ func (service *VPCService) ListVPCInfo(ns string) []common.VPCResourceInfo {
 	return VPCInfoList
 }
 
-func (s *VPCService) GetNSXLBSPath(lbsId string) string {
```

Comment:
Add some comments here to explain why the caller shall provide the VPC's ID but not the LBS's ID?

Personally, I would more like to rename this function as `GetDefaultNSXLBSPathByVPC`, then we can have other functions to support getting NSX lb path directly by LBS's ID in future if multiple LBS are required in one VPC. Not strong opinion at this moment because we now only have one lbs per VPC.